documentation/service-rules/school-admins.md

```diff
@@ -2,89 +2,103 @@
 
-We should always ask for two fields of personal information before registering an ECT's record. This is because we need to make sure the person registering the ECT actually knows that ECT. If we just ask for TRN, they could be entering a random number.
+To do this, we ask for the ECT’s TRN and date of birth.
 
-If we can't find a matching TRN that exists, we tell the user the ECT's teacher record cannot be found.
+We should always ask for two fields of personal information before registering an ECT’s record. This is because we need to make sure the person registering the ECT actually knows that ECT. If we just ask for TRN, they could be entering a random number.
 
-If we can't find the ECT's record in the Teaching Record System API, but the TRN does exist, we ask for that ECT's national insurance number instead. We do this because potentially the date of birth stored on the teacher's record in the TRS is incorrect, or the one the school holds might be. It makes it more likely for the user to find the ECT's teacher record and confirm they know them.
+If we can’t find a matching TRN that exists, we tell the user the ECT’s teacher record cannot be found.
+
+If we can’t find the ECT’s record in the Teaching Record System API, but the TRN does exist, we ask for that ECT’s national insurance number instead. We do this because potentially the date of birth stored on the teacher’s record in the TRS is incorrect, or the one the school holds might be. It makes it more likely for the user to find the ECT’s teacher record and confirm they know them.
 
 If we still cannot confirm the user knows both the TRN and either national insurance number or date of birth, we tell them the record cannot be found.
 
-Neither date of birth or national insurance number should be stored longer-term. It is just used for the initial finding and checking of an ECT's record in the TRS. You can read more [about our reasoning for this here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/no-longer-storing-date-of-birth/).
+Neither date of birth or national insurance number should be stored longer-term. It is just used for the initial finding and checking of an ECT’s record in the TRS. You can read more [about our reasoning for this here](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/no-longer-storing-date-of-birth/).
 
 ### Checking the ECT is eligible to be registered for training
 
 When an ECT is registered for training, we should check if their record already exists in CPD.
 
-If the ECT being registered already exists as a mentor, we should not let them progress with registration. This is because a mentor undergoing ECT training is highly unlikely and it's probably a mistake.
+If the ECT being registered already exists as a mentor, we should not let them progress with registration. This is because a mentor undergoing ECT training is highly unlikely and it’s probably a mistake.
```

Comment:
```suggestion
If the ECT already exists registered as a mentor, we should not let them progress with registration. This is because a mentor undergoing ECT training is highly unlikely and it’s probably a mistake.
```